### ISSUE TITLE
create login

### ISSUE THREAD

**Issue by harshraj22** [harshraj22/contest#12]
Create a login txt/php/other file. This contains space for login/password/other details for local mysql system. After this, add it to gitignore so that contributors can add their credentials without sharing it publically and all sql queries use this variable.

**Comment by DestroyerAlpha** [comment]
Do you mean like a signup page?

**Comment by harshraj22** [comment]
No, for accessing mysql, mysqli_connect needs user name and password for local server. Now everyone has these credentials different, so everyone fills their credentials in this page. Adding it to gitignore makes sure these details are not pushed. Now mysqli_connect uses these variables, present in this page to connect. so that every-one connects on their server easily without sharing credentials.

**Comment by abhiisshheekk** [comment]
working on it

**Comment by DestroyerAlpha** [comment]
I guess this can be closed now!
